CHANGELOG.md

```diff
@@ -2,7 +2,42 @@
 All notable changes to this project will be documented in this file.
 See the [change log guidelines](http://keepachangelog.com/) for information on how to structure the file.
 
-## [Unreleased]
+## [5.1.0]
+
+### Added
+- Support for reading multiple sheets from an Excel file simultaneously
+- Enable exporting of multiple feature types to Excel
+- Allow loading Excel with blank rows and sheets for schema and source data
+- Add predefined and custom formatters for Date cells in Excel import
+- Enable exporting feature types with no data in Excel
+- Align Excel with CSV export wizard
+- Prefer properties with short paths as GeoJSON geometry property
+- Add wizard page to select the read mode for GeoJSON/JSON
+- Extend JSON reader functionality
+- Support loading schema from JSON data
+- Configure decimal places in JSON
+- Export valid Shapefiles for mappings, including feature types with no geometry
+- Support downloading WFS files to the tmp folder for bucket service
+- Feature: Type selection broken on custom export config
+- Support adding multiple selections of feature types to be exported to CSV
+- Replace default map with OpenStreetMap
+- Add Boolean and Java Pattern class to Groovy restriction whitelist
+- Update decimal places to 20 for GeoTools
+
+## Maintenance
+- Update Geotools to version 29.1
+- Update dependencies in hale-platform to resolve security issues
+- Fix for the Windows installer
+ 
+### Fixed
+- Fix Excel reader not adding instances for empty rows
```

Comment:
So now instances are added for empty rows?

Reply:
NO instance is added if excel contains an empty row.

Reply:
To me fixing "not adding" sounds like now they are added. Maybe something like this would be more clear:

"Don't create instances from empty rows"